lib/public/User/Events/UserConfigChangedEvent.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023 Murena SAS <akhil.potukuchi.ext@murena.com>
+ *
+ * @author Murena SAS <akhil.potukuchi.ext@murena.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCP\User\Events;
+
+use OCP\EventDispatcher\Event;
+
+/**
+ * @since 31.0.0
```

Comment:
```suggestion
 * @since 32.0.0
```
I’m sorry about the running gag, but this needs to be bumped to 32 now :see_no_evil: (and all the other `@since` as well).
Also, it would be good to add the new attribute `Listenable`, see https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/api.html#php-public-api